datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -309,6 +309,7 @@ private StackV4Request getStackRequest(SdxClusterRequest sdxClusterRequest, Stac
                         ", runtime version: " + runtimeVersion);
             }
             stackRequest.getCluster().setRangerRazEnabled(sdxClusterRequest.isEnableRangerRaz());
+            stackRequest.setImage(sdxClusterRequest.getImageSettingsV4Request());
```

Comment:
What if somebody uses internal api and sends a stackv4request without image info and would like to use image provided in image field? It should be handled in 'else' branch.

Reply:
OK. I will handle that scenario as well. Thanks!

Reply:
Handle that for both cases